backend/Dockerfile.launcher

```diff
@@ -37,7 +37,7 @@ RUN GO111MODULE=on go mod download
 COPY ${SOURCE_CODE}/ ./
 
 RUN GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -tags netgo -ldflags '-extldflags "-static"' -o /bin/launcher-v2 ./backend/src/v2/cmd/launcher-v2/*.go
-RUN GO111MODULE=on CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GOEXPERIMENT=strictfipsruntime go build -tags 'netgo strictfipsruntime' -o /bin/launcher-v2-fips ./backend/src/v2/cmd/launcher-v2/*.go
+RUN GO111MODULE=on CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GOEXPERIMENT=strictfipsruntime go build -tags strictfipsruntime -o /bin/launcher-v2-fips ./backend/src/v2/cmd/launcher-v2/*.go
```

Comment:
I see we use `netgo` for the non fips version, but removed it for the cgo enabled version, what's the reason for that?

Reply:
same question for launcher